src/terms/ewald.jl

```diff
@@ -33,31 +33,39 @@ function energy_ewald(model::Model{T}; kwargs...) where {T}
     energy_ewald(model.lattice, charges, model.positions; kwargs...)
 end
 
+# We balance the Ewald spling of ``1/r ≡ erf(η·r)/r + erfc(η·r)/r`` between reciprocal
+# summation and real-space summation with a slight bias towards reciprocal summation.
+function default_η(lattice::AbstractArray{T}) where {T}
+    any(iszero.(eachcol(lattice))) && return  # We won't compute anything
+    recip_lattice = compute_recip_lattice(lattice)
+    sqrt(sqrt(T(1.69) * norm(recip_lattice ./ 2T(π)) / norm(lattice))) / 2
+end
+
 # This could be merged with Pairwise, but its use of `atom_types` would slow down this
 # computationally intensive Ewald sums. So we leave it as it for now.
 """
-Compute the electrostatic interaction energy per unit cell between point charges in
-a uniform background of compensating charge to yield net neutrality.`lattice` should
-contain the lattice vectors as columns. `charges` and `positions` are the point charges and
-their positions (as an array of arrays) in fractional coordinates. If `forces` is not
-nothing, minus the derivatives of the energy with respect to `positions` is computed.
+Compute the electrostatic interaction energy per unit cell between point charges at
```

Comment:
This doc has to be changed. In the phonon case, the energy per unit cell does not make sense, right? Similarly, the forces are the forces on the atoms of the first unit cell?

Reply:
Ah, we converged on a comment in pairwise.jl, so just copy paste it here?